lib/koala/api.rb

```diff
@@ -77,7 +77,13 @@ def api(path, args = {}, verb = "get", options = {}, &error_checking_block)
 
         # if we want a component other than the body (e.g. redirect header for images), return that
         if component = options[:http_component]
-          component == :response ? result : result.send(options[:http_component])
+          if component == :response
+            parsed_body = MultiJson.load("[#{result.body.to_s}]")[0]
```

Comment:
Koala's no longer using MultiJson -- there's an example JSON.parse line that you can copy below in the updated version.